### PR TITLE
VTCCA-6651 Play 3.0 Upgrade

### PR DESCRIPTION
**Pull request by WillBlakeney**

### CODE REVIEW

**Comment by WillBlakeney** [comment]
The tests should pass once https://github.com/hmrc/build-jobs/pull/17134 is merged

**Comment by WillBlakeney** [comment]
rebuild

File: app/models/modernised/PropertyLinkStatus.scala [hunk 1]
```diff
@@ -27,5 +29,5 @@ object PropertyLinkStatus extends Enumeration {
   val REVOKED = Value("REVOKED")
   val MORE_EVIDENCE_REQUIRED = Value("MORE_EVIDENCE_REQUIRED")
 
-  implicit val format = enumFormat(PropertyLinkStatus)
+  implicit val format: Format[modernised.PropertyLinkStatus.Value] = enumFormat(PropertyLinkStatus)
```
Comment (zachhall1234) [inline app/models/modernised/PropertyLinkStatus.scala:32]: Not a functional issue but could this just be Format[PropertyLinkStatus.Value]?

Comment (WillBlakeney) [reply app/models/modernised/PropertyLinkStatus.scala]: Yeah it definitely can, I must've missed that 👍

**Review by zachhall1234 — APPROVED** [review]

**Pull request merged**